comprobo_final_project/scripts/models/robot.py

```diff
@@ -15,29 +15,37 @@ def __init__(self):
         # TODO: This needs some work, b/c we can't start multiple robot nodes here.
         rospy.init_node('robot_controller')
 
-        self.pose = PoseStamped()
+        self.pose_stamped = PoseStamped()
         self.twist = Twist()
+        self.resolution = 100
 
         # Suscribe to position of Neato robot
-        # GazeboPoseProvider(rospy).subscribe(self._pose_listener) # For Gazebo
-        AprilPoseProvider(rospy).subscribe(self._pose_listener) # For real world
+        GazeboPoseProvider(rospy).subscribe(self._pose_listener) # For Gazebo
+        # AprilPoseProvider(rospy).subscribe(self._pose_listener) # For real world
```

Comment:
The difference between Gazebo/Real World should be stored in a variable within Robot class whenever a Robot object is created.